outputs/alertmanager_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/falcosecurity/falcosidekick/types"
 )
 
+const defaultThresholds = `[{"priority":"critical", "value":10000}, {"priority":"critical", "value":1000}, {"priority":"critical", "value":100} ,{"priority":"warning", "value":10}, {"priority":"warning", "value":1}]`
```

Comment:
I'm using json inside tests due to parsing is limited to the config.go file, so, to avoid changing the structure and to avoid duplicating code, I added the threshold as Unmarshal Json.

Reply:
ok